test/case/ietf_interfaces/remove_ip_address.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+import infamy
+
+#Name of the interface to remove IPv4 addresses from:
+interface_name = "eth0"
```

Comment:
According to your suggestion below: Expand `remove_ip_address` by first adding the address, like this test does, and then rename the case to `ipv4_address `or similar. If we do this, do we need to dynamically discover it since we'll add an address to a certain interface and we'll have that information afterwards. Or this is something to be changed in both test cases (`set_ip_address` and `remove_ip_address`) so we firstly search for available interfaces?